src/vorta/utils.py

```diff
@@ -191,35 +196,31 @@ def get_private_keys():
             key_file = os.path.join(ssh_folder, key)
             if not os.path.isfile(key_file):
                 continue
-            for key_format in key_formats:
-                try:
-                    parsed_key = key_format.from_private_key_file(key_file)
+            # ignore config, known_hosts*, *.pub, etc.
+            if key.endswith('.pub') or key.startswith('known_hosts') or key == 'config':
+                continue
+            try:
+                if is_ssh_file(key):
                     key_details = {
                         'filename': key,
-                        'format': parsed_key.get_name(),
-                        'bits': parsed_key.get_bits(),
-                        'fingerprint': parsed_key.get_fingerprint().hex(),
                     }
                     available_private_keys.append(key_details)
-                except (
-                    SSHException,
-                    UnicodeDecodeError,
-                    IsADirectoryError,
-                    IndexError,
-                    ValueError,
-                    PermissionError,
-                    NotImplementedError,
-                ):
-                    logger.debug(
-                        f'Expected error parsing file in .ssh: {key} (You can safely ignore this)', exc_info=True
-                    )
+            except (
+                UnicodeDecodeError,
+                IsADirectoryError,
+                IndexError,
+                ValueError,
+                PermissionError,
+                NotImplementedError,
+            ):
```

Comment:
Do we need to catch all of those errors?

Reply:
yeah, I needed some context about why they were here at the first place, thanks! I will remove `NotImplementedError` and `IsADirectoryError`

Reply:
Probably some more to remove.